sonic-net/sonic-utilities
# [Kdump] Fix the error message when trying to enable kdump if kdump was already enabled

#### What I did
The following error message will be shown in the syslog by doing "sudo config kdump enable" when kdump was already enabled.
`INFO hostcfgd[6930]: Error Unable to unload Kdump the kernel '%s' []`
`INFO hostcfgd[6930]: kdump is already enabled`
`ERR hostcfgd: sonic-kdump-config --enable - failed: return code - 1, output:#012None`

The root cause is that there has some logical error in 'kdump_enable()' function
#### How I did it
Modified the logic in kdump_enable() which will return directly
when there does not need to apply any change after comparing to the expected
configuration and the current operational mode.
#### How to verify it
1. Make sure kdump is already enabled.
2. Do "sudo config kdump enable" again
3. The error message will show in the syslog
#### Previous command output (if the output of a command-line utility has changed)
No command-line changed
#### New command output (if the output of a command-line utility has changed)
No command-line changed
